src/GovukNotify/Models/CostDetails.cs

```diff
@@ -0,0 +1,42 @@
+#nullable enable
+
+using Newtonsoft.Json;
+
+namespace Notify.Models
+{
+    public class CostDetails
+    {
+        [JsonProperty("billable_sms_fragments")]
+        public int? billableSmsFragments;
+
+        [JsonProperty("international_rate_multiplier")]
+        public double? internationalRateMultiplier;
+
+        [JsonProperty("sms_rate")]
+        public double? smsRate;
+
+        [JsonProperty("billable_sheets_of_paper")]
+        public int? billableSheetsOfPaper;
+
+        public string? postage;
+
+        public override bool Equals(object? other)
+        {
+            if (other is not CostDetails o)
+            {
+                return false;
+            }
+
+            return billableSmsFragments == o.billableSmsFragments &&
+                   internationalRateMultiplier == o.internationalRateMultiplier &&
+                   smsRate == o.smsRate &&
+                   billableSheetsOfPaper == o.billableSheetsOfPaper &&
+                   postage == o.postage;
+        }
+
+        public override int GetHashCode()
+        {
+            return base.GetHashCode();
```

Comment:
Maybe not one for this PR but I think we should probably implement GetHashCode properly, throughout the notify client